Samples/SampleWdpClient.UniversalWindows/MainPage.xaml.cs

```diff
@@ -113,7 +113,7 @@ private void ConnectToDevice_Click(object sender, RoutedEventArgs e)
                         // remainder of this session.
                         if (allowUntrusted)
```

Comment:
I think we can change this allowUntrusted to be "allowUntrusted || \<is using http instead of https (eg USB)\> ?